src/BWFMetaEditBundle/Resources/views/Download/fedoraTable.html.twig

```diff
@@ -2,26 +2,38 @@
     <th colspan="4">For automatic update and easiest installation, enable MediaArea software repository for Fedora<br>search for "BWF MetaEdit" in your package manager<br>or "yum install bwfmetaedit-gui" (for GUI)<br> or "yum install bwfmetaedit" (for CLI)</th>
     <td><a href="{{ path('ma_repos') }}">Activate MediaArea Repository</a></td>
 </tr>
+<tr>
+    <th rowspan="2" id="41">41</th>
+    <th rowspan="2" id="41.x86_64">x86_64</th>
+    <td><abbr title="Graphical User Interface">GUI</abbr></td>
+    <td><a href="//mediaarea.net/download/binary/bwfmetaedit-gui/25.03/bwfmetaedit-gui-25.03.x86_64.Fedora_41.rpm">v25.03</a></td>
+    <td>&nbsp;</td>
+</tr>
+<tr>
+    <td><abbr title="Command Line Interface">CLI</abbr></td>
+    <td><a href="//mediaarea.net/download/binary/bwfmetaedit/25.03/bwfmetaedit-25.03.x86_64.Fedora_41.rpm">v25.03</a></td>
+    <td>&nbsp;</td>
+</tr>
 <tr>
     <th rowspan="2" id="40">40</th>
     <th rowspan="2" id="40.x86_64">x86_64</th>
     <td><abbr title="Graphical User Interface">GUI</abbr></td>
-    <td><a href="//mediaarea.net/download/binary/bwfmetaedit-gui/24.10/bwfmetaedit-gui-24.10.x86_64.Fedora_40.rpm">v24.10</a></td>
+    <td><a href="//mediaarea.net/download/binary/bwfmetaedit-gui/25.03/bwfmetaedit-gui-25.03.x86_64.Fedora_40.rpm">v25.03</a></td>
     <td>&nbsp;</td>
 </tr>
 <tr>
     <td><abbr title="Command Line Interface">CLI</abbr></td>
     <td><a href="//mediaarea.net/download/binary/bwfmetaedit/24.10/bwfmetaedit-24.10.x86_64.Fedora_40.rpm">v24.10</a></td>
```

Comment:
[bwfmetaedit-25.03.x86_64.Fedora_40.rpm](https://mediaarea.net/download/binary/bwfmetaedit/25.03/bwfmetaedit-25.03.x86_64.Fedora_40.rpm) exists but not referenced here.